### ISSUE TITLE
Is there any way to disable custom ktlint rule?

### ISSUE THREAD

**Issue by imbyungjun** [pinterest/ktlint#1287]
## Expected Behavior

Custom ktlint rule can also be disabled with `// ktlint-disable my-rule`.

## Current Behavior

Cannot disable custom ktlint rule with giving a specific rule-id. 
I know that using `/* ktlint-disable */` works fine but, would be happy if I can disable it with specific rule-id

## Additional information
* Current version of ktlint: 0.43.0

**Comment by romtsn** [comment]
Yep, you also need to specify your ruleset id, something like:
```Kotlin
// ktlint-disable custom-ruleset:my-rule
```
then it should work
